src/pieces/commandManager.ts

```diff
@@ -85,6 +85,10 @@ async function runCommand(msg: Message) {
 
 	const command = getCommand(msg.client, commandName);
 	if (!command || command.enabled === false) return;
+	if (msg.channel.id !== CHANNELS.SAGE && command.restricted) {
```

Comment:
This logic ignores DMs. Commands that can run in DMs (like submit) probably should still be able to run in DMs.

Reply:
good catch sir